tracer/build/_build/docker/smoke.windows.dd-dotnet.dockerfile

```diff
@@ -0,0 +1,55 @@
+﻿ARG DOTNETSDK_VERSION
+ARG RUNTIME_IMAGE
+
+# Build the ASP.NET Core app using the latest SDK
+FROM mcr.microsoft.com/dotnet/sdk:$DOTNETSDK_VERSION as builder
+
+# Build the smoke test app
+WORKDIR /src
+COPY ./test/test-applications/regression/AspNetCoreSmokeTest/ .
+
+ARG PUBLISH_FRAMEWORK
+RUN dotnet publish "AspNetCoreSmokeTest.csproj" -c Release --framework %PUBLISH_FRAMEWORK% -o /src/publish
+
+FROM $RUNTIME_IMAGE AS publish-msi
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+WORKDIR /app
+
+ARG CHANNEL_32_BIT
+RUN if($env:CHANNEL_32_BIT){ \
+    echo 'Installing x86 dotnet runtime ' + $env:CHANNEL_32_BIT; \
+    curl 'https://dot.net/v1/dotnet-install.ps1' -o dotnet-install.ps1; \
+    ./dotnet-install.ps1 -Architecture x86 -Runtime aspnetcore -Channel $env:CHANNEL_32_BIT -InstallDir c:\cli; \
+    [Environment]::SetEnvironmentVariable('Path',  'c:\cli;' + $env:Path, [EnvironmentVariableTarget]::Machine); \
+    rm ./dotnet-install.ps1; }
+
+# Copy the installer files from tracer/test/test-applications/regression/AspNetCoreSmokeTest/artifacts
+COPY --from=builder /src/artifacts /install
+
+RUN mkdir /logs; \
+    cd /install; \
+    Start-Process -Wait msiexec -ArgumentList '/qn /i datadog-apm.msi'; \
+    cd /app; \
+    rm /install -r -fo
+
+# Set the additional env vars
+ENV DD_PROFILING_ENABLED=1 \
+    DD_APPSEC_ENABLED=1 \
+    DD_TRACE_DEBUG=1 \
+    CORECLR_ENABLE_PROFILING=1 \
+    CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8} \
```

Comment:
I think you shouldn't set those? dd-dotnet is supposed to set them.
Also, I think you should kill two birds with one stone and use `--set-env` in dd-dotnet to set the other variables

Reply:
I switched some of the extra env vars to use `--set-env`, but it gets a bit unwieldy so left some of them